<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfiguratorGeneralDataWidget</class>
 <widget class="QWidget" name="ConfiguratorGeneralDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General data editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QGroupBox" name="tableBox">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="countLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <row>
         <property name="text">
          <string>Новая строка</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Новая строка</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Новая строка</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="tableListBox">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Tables</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
